smoke-tests/fake-backend/src/main/java/io/opentelemetry/smoketest/fakebackend/FakeBackendMain.java

```diff
@@ -69,25 +70,38 @@ public void serialize(
           }
         });
     module.setSerializers(serializers);
+    serializers.addSerializer(
```

Comment:
Extreme nit, looks a bit nicer to do this before `setSerializers`

Reply:
Whoops, thanks, this was a mistake to have it in there twice.  Copypasta.  Will fix.  🎈